frontend/src/metabase/public/components/EmbedFrame.css

```diff
@@ -38,3 +38,16 @@
 .Theme--night.EmbedFrame .enable-dots .dc-tooltip circle.dot {
   fill: currentColor;
 }
+
+.Theme--transparent.EmbedFrame {
+  background-color: transparent;
+}
+
+.Theme--transparent .EmbedFrame-header,
+.Theme--transparent .EmbedFrame-footer {
+  background-color: transparent;
+}
+
+.Theme--transparent.EmbedFrame .DashCard .Card {
+  background-color: transparent;
+}
```

Comment:
This line is for the dashboard card.
![image](https://user-images.githubusercontent.com/1937582/175048335-fbd1bb5b-7caa-401d-a673-f1edba9c2029.png)